content-src/components/Search/_Search.scss

```diff
@@ -137,6 +131,39 @@ $glyph-forward: url('chrome://browser/skin/forward.svg');
   }
 }
 
+.below-search-snippet {
+  margin: 0 auto;
+  width: $searchbar-width-small;
+
+  @media (min-width: $break-point-medium) {
+    width: $searchbar-width-medium;
+  }
+
+  @media (min-width: $break-point-large) {
+    width: $searchbar-width-large;
+  }
+}
+
+.non-collapsible-section + .below-search-snippet {
+  // If search is enabled, we need to invade its large bottom padding.
+  margin-top: -44px;
```

Comment:
Personally I think this looks right around `-50px` but might be good to check with wolasi (feel free to land and we can follow-up with final tweaks)